modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/EpoExtended.pm

```diff
@@ -316,7 +309,7 @@ sub pipeline_analyses_db_complete {
     return [
         # ---------------------------------------------------[Delete base alignment]-----------------------------------------------------
         {   -logic_name => 'delete_alignment',
-            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::EpoLowCoverage::DeleteEPO',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::EpoExtended::DeleteEPO',
```

Comment:
`DeleteEPO` is expecting a `base_epo_mlss_id` param. You could change that in the runnable itself, but github won't let me suggest changes on lines you haven't touched 🙄